lib/codecs/src/encoding/format/csv.rs

```diff
@@ -112,7 +310,7 @@ mod tests {
 
```

Comment:
Please add a test case for `QuoteStyle::NonNumeric`

Reply:
Agreed. I added tests for all the cases in `correct_quoting()`